### ISSUE TITLE
unittest_windows_gpu_py3.8  CI failing

### ISSUE THREAD

**Issue by prabhat00155** [pytorch/vision#4560]
"torch.cuda.is_available() is False"
```
+ '[' 3.8 == 3.6 ']'
++ python -c 'import torch; print(torch.cuda.is_available())'
+ torch_cuda=False
+ echo torch.cuda.is_available is False
torch.cuda.is_available is False
+ '[' '!' -z 11.1 ']'
+ '[' False == False ']'
+ echo 'torch with cuda installed but torch.cuda.is_available() is False'
torch with cuda installed but torch.cuda.is_available() is False
+ exit 1
```

https://app.circleci.com/pipelines/github/pytorch/vision/11101/workflows/fe23d326-ed62-4477-86b6-61432ca4c238/jobs/854690

https://app.circleci.com/pipelines/github/pytorch/vision/11099/workflows/5aaea50a-e67f-4770-9e9e-f212d39733d8/jobs/854513

cc @peterjc123 @nbcsm @guyang3532 @maxluk @gunandrose4u @mszhanyi @seemethere

**Comment by oke-aditya** [comment]
This has occurred once again.

https://app.circleci.com/pipelines/github/pytorch/vision/11304/workflows/14a92f57-9098-4519-abbc-48b5ad8cb8b2/jobs/874287

**Comment by mszhanyi** [comment]
@prabhat00155 @oke-aditya It has been fixed by #4788
If you came across again, please feel free to reopen it.
